sdk/cosmos/azure-cosmos/test/test_auto_scale.py

```diff
@@ -18,36 +18,36 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-import unittest
-import azure.cosmos.cosmos_client as cosmos_client
+
+from azure.cosmos import CosmosClient
 import azure.cosmos.exceptions as exceptions
-from azure.cosmos import ThroughputProperties, PartitionKey, http_constants
+from azure.cosmos import ThroughputProperties, PartitionKey
 import pytest
 import test_config
 
 pytestmark = pytest.mark.cosmosEmulator
 
 
 @pytest.mark.usefixtures("teardown")
-class AutoScaleTest(unittest.TestCase):
+class TestAutoScale:
     host = test_config._test_config.host
     masterKey = test_config._test_config.masterKey
     connectionPolicy = test_config._test_config.connectionPolicy
 
     @classmethod
-    def setUpClass(cls):
+    def _set_up(cls):
         if (cls.masterKey == '[YOUR_KEY_HERE]' or
                 cls.host == '[YOUR_ENDPOINT_HERE]'):
             raise Exception(
                 "You must specify your Azure Cosmos account values for "
                 "'masterKey' and 'host' at the top of this class to run the "
                 "tests.")
 
-        cls.client = cosmos_client.CosmosClient(cls.host, cls.masterKey, consistency_level="Session",
-                                                connection_policy=cls.connectionPolicy)
+        cls.client = CosmosClient(cls.host, cls.masterKey, consistency_level="Session")
```

Comment:
Why are we skipping connectionPolicy parameter?

Reply:
the connection policy that was previously there is outdated and had default settings anyway